### ISSUE TITLE
The current version of drop-in hangs at loading

### ISSUE THREAD

**Issue by juongithub** [braintree/braintree-web-drop-in#122]
### General information

* SDK/Library version: unreleased (according to changelog)
* Environment: Sandbox (I have not tested it on production)
* Chrome  57.0.2987.133, Firefox 52.0.2 (32 bit), IE 11 on Windows 8.1 (64bit)

### Issue description

The dropb-in hangs at "loading" showing the loading throbber indefinitely. 

I'm using the initialization script:
```
<script>
	var submitButton = document.querySelector('#submit-button');

	braintree.dropin.create({
		authorization: '[CLIENT_AUTHORIZATION]',
		selector: '#dropin-container'
	}, function (err, dropinInstance) {
		if (err) {
			// Handle any errors that might've occurred when creating Drop-in
			console.error(err);
			return;
		}
		submitButton.addEventListener('click', function () {
			dropinInstance.requestPaymentMethod(function (err, payload) {
				if (err) {
					console.log(err);

					return;
				}

				alert(payload);
			});
		});
	});
</script>
```
I have tried the cases:
```
1. [CLIENT_AUTHORIZATION]=Braintree_ClientToken::generate()
2. [CLIENT_AUTHORIZATION]=Braintree_ClientToken::generate(array(
			'customerId' => $customer_id,
			'options'    => array(
				'verifyCard'                   => true,
				'failOnDuplicatePaymentMethod' => true
			)
		));
```

Please note that the previous version works correctly (with the same settings).

**Comment by juongithub** [comment]
Please note that the previous version works correctly (with the same settings).

**Comment by lilaconlee** [comment]
This is likely because you are using a locally built version of Drop-in JS, but the CSS included on your page is our hosted CSS from version 1.0.0-beta.6. If you’re building Drop-in yourself, you will need to include the built stylesheet with a `braintree-dropin-stylesheet` id on your page to overwrite the hosted version. This might look like:
```html
<link rel="stylesheet" type="text/css" href="/path/to/custom/dropin.css" id="braintree-dropin-stylesheet">
```

This being said, while we try our best to keep `master` fully functional, we do not guarantee that will always be the case and recommend using an official release.

**Comment by juongithub** [comment]
Sorry, but your suggestion does not solve the problem...

**Comment by crookedneighbor** [comment]
@juongithub Are you getting any console errors?

**Comment by juongithub** [comment]
No error on the console:
1. <link rel="stylesheet" type="text/css" href="css/dropin.css" id="braintree-dropin-stylesheet"><script src="js/dropin.js"></script> => hangs
2. <script src="js/dropin.js"></script> => hangs
3. <link rel="stylesheet" type="text/css" href="css/dropin.css" id="braintree-dropin-stylesheet">
<script src="https://js.braintreegateway.com/web/dropin/1.0.0-beta.6/js/dropin.min.js"></script> => does not hang
4. <script src="https://js.braintreegateway.com/web/dropin/1.0.0-beta.6/js/dropin.min.js"></script> => does not hang

**Comment by crookedneighbor** [comment]
@juongithub So far we have not been able to reproduce it. Can you provide us with a staging site or a Github repo that reproduces the issue?

**Comment by crookedneighbor** [comment]
Feel free to email web-drop-in-beta@braintreepayments.com if you do not want to post information about a staging site here.

**Comment by juongithub** [comment]
You can see the files I'm using here:
https://github.com/juongithub/braintree-web-drop-in/tree/master/issue_122

**Comment by juongithub** [comment]
Is it possible to be caused by a "build" problem on my platform?

**Comment by juongithub** [comment]
Maybe this could be useful, here is the list of network events both for "working" and "not working":
![image](https://cloud.githubusercontent.com/assets/21062625/24883076/aa470f32-1e43-11e7-9ca4-b93efdd73a9f.png)

**Comment by juongithub** [comment]
Ok, I finally managed to solve this issue... Actually, the problem was caused by concomitant problems:
1. the current version of CSS is not compatible with the unreleased dropin: the current dropin (i.e., https://js.braintreegateway.com/web/dropin/1.0.0-beta.6/js/dropin.min.js) loads correctly only when using the current CSS (i.e., https://assets.braintreegateway.com/web/dropin/1.0.0-beta.6/css/dropin.min.css).

2. I had an old version of main.scss in my repository: this caused the failing of other combinations.
